kiwix-build_ci/noble_builder.dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM ubuntu:noble
+LABEL org.opencontainers.image.source https://github.com/kiwix/container-images
+
+ENV LANG C.UTF-8
+ENV OS_NAME numbat
```

Comment:
noble ?

Reply:
Yep ; other ones are `bionic` and `focal` so it should be `noble`

Reply:
Fixed